### PR TITLE
SA2B: Update Setup guide for new Mod Manager

### PR DESCRIPTION
**Pull request by PoryGone**
## What is this fixing or adding?
SA2 now has a fancy new Mod Manager, and the old one is deprecated. This requires slightly different setup instructions.

Additionally, a single, much-requested option tooltip clarification is included.

## How was this tested?
We read it, and followed it, and it worked.

### CODE REVIEW

**Review by nicholassaylor — APPROVED** [review]
Read through the updated guide and checked all links. Guide was easy to follow
Did not install SA2B or the mod manager as part of reading
LGTM

File: worlds/sa2b/Options.py [hunk 1]
```diff
@@ -227,7 +227,8 @@ class Omosanity(Toggle):
 
 class Animalsanity(Toggle):
     """
-    Determines whether picking up counted small animals grants checks
+    Determines whether unique counted animals grants checks.
```
Comment (Exempt-Medic) [inline worlds/sa2b/Options.py:230]: I don't really know what "unique counted" is meant to mean here. Maybe just "unique" or "uniquely named"?

Comment (Exempt-Medic) [reply worlds/sa2b/Options.py:230]: So can it not just be "Determines whether animals grant checks"?

Comment (Exempt-Medic) [reply worlds/sa2b/Options.py:230]: You could use "unique counts/totals/numbers of animals grant checks"

Comment (RaspberrySpace) [reply worlds/sa2b/Options.py:230]: I would be fine with "Determines whether unique counts of animals grants checks."

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**